packages/eslint-plugin/docs/rules/no-magic-numbers.md

```diff
@@ -13,14 +13,18 @@ It adds support for:
 
 ```jsonc
 {
-  // note you must disable the base rule as it can report incorrect errors
-  "no-magic-numbers": "off",
-  "@typescript-eslint/no-magic-numbers": [
-    "error",
-    {
-      /* options */
-    }
-  ]
+  ...,
+  "rules: {
```

Comment:
Invalid syntax, missing `"`. Also, `...,` should be `// ...`

Everywhere else, the options example is this format (only `rules`). e.g. https://typescript-eslint.io/rules/no-shadow. Should we both add a [code block title](https://docusaurus.io/docs/next/markdown-features/code-blocks#code-title) and use this more complete format?

Reply:
Fair point.. apologies for the poor syntax there.

I came here from Google (just searched for some logic I wanted and click through to this specific readme), I wasn't navigating through the code-base with context. Perhaps a note at the top of each README would help understand?

I only lost a few minutes time, but for someone implementing for the first time, it's not immediately obvious.

Reply:
As for the code block title I'm not sure... anything that would indicate this config goes inside the `rules` sub-object (rather than looking like a top level obj) wouldn't hurt imo..